README.rst

```diff
@@ -55,6 +55,17 @@ extendable and language independent. coala is written with a lower case
 
 Read more at our `documentation <http://coala.rtfd.org/>`__.
 
+Why use coala?
+--------------
+
+- coala provides built-in checking routines (named bears in coala).
+- Serves your requirement: You can easily write your own checks (using bears).
+- coala provides user-friendly interfaces like json, formatted and interactive
+  output in the CLI and plugins for various editors are available as well.
+- Optimal Performance: coala manages parallelizing the checking-routines
+  without you having to worry.
+- Unified Interface: One tool to take care of other tools.
```

Comment:
Yup highlighting the catch phrases is good idea, may be in all items , like : 
![screenshot from 2016-03-16 17-43-07](https://cloud.githubusercontent.com/assets/8680142/13811894/a70f0d66-eb9e-11e5-8e01-61fe5c8b3138.png)